includes/Main.php

```diff
@@ -1447,25 +1447,36 @@ public function log_document_creation_trigger_to_order_meta( $document, $trigger
 	/**
 	 * Get the document triggers
 	 *
-	 * @param bool $only_manual_triggers
+	 * @param string $trigger_type
```

Comment:
Would it be helpful to add a brief description here outlining the available options?

Reply:
Nice idea! Thanks